preview-env/comment/action.yml

```
@@ -0,0 +1,39 @@
+---
+name: Create Deployment Result Summary
+description: |
+  Creates a deployment result summary as comment in the current PR when deployment deployment artifacts have been uploaded during this run.
```

The aim of this composite action is to manage comments (create, update, and probably deletion ?), so the name should reflect it.

```suggestion
name: Manage Deployment Result Summary
description: |
  Manages a deployment result summary as comment in the current PR
```